devops/jobs/CreateSandboxCI.groovy

```diff
@@ -61,8 +61,6 @@ class CreateSandboxCI {
                   booleanParam('reconfigure',true)
                   booleanParam('recreate',false)
                   predefinedProp('name_tag','continuous-integration')
-                  booleanParam('xqueue',true)
-                  booleanParam('xserver',true)
```

Comment:
You want to move these up into the Daily one.  They default to false on the sandbox job (so a move rather than a delete)